pkg/terraform/terraform.go

```diff
@@ -20,21 +20,26 @@ const DefaultWorkingDir = "/cnab/app/terraform"
 
 const defaultTerraformVersion = "0.12.17"
 
+// DefaultTerraformVarFilename is the default name for terrafrom tfvars json file
+const DefaultTerraformVarFilename = "terraform.tfvars.json"
```

Comment:
Assuming the creation of this file will always be done in the install step, can we move its definition to install.go and remove it from the `Mixin` struct here?